### PR TITLE
jsonrpc macros

### PR DESCRIPTION
**Pull request by debris**

### CODE REVIEW

File: macros/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[package]
+description = "Helper macros for jsonrpc-core"
+homepage = "https://github.com/ethcore/jsonrpc"
+repository = "https://github.com/ethcore/jsonrpc"
+license = "MIT"
+name = "jsonrpc-macros"
+version = "0.1.0"
+authors = ["debris <marek.kotewicz@gmail.com>"]
```
Comment (rphmeier) [inline macros/Cargo.toml]: oh, is that so? ;)

**Comment by rphmeier** [comment]
LGTM

**Comment by tomusdrw** [comment]
Would be cool to get some examples

**Comment by debris** [comment]
travis is stuck ;(

**Pull request merged**